packit/cli/dist_git_init.py

```diff
@@ -43,7 +44,10 @@
 @click.option(
     "--upstream-git-url",
     help="URL to the upstream GIT repository",
-    required=True,
+)
+@click.option(
+    "--upstream-git-url-command",
+    help="Command to get a URL of the upstream GIT repository",
```

Comment:
```suggestion
    help="Command to get the URL of the upstream GIT repository",
```

Reply:
```suggestion
    help="Command to get the URL of the upstream git repository",
```

Sorry, but I don't like it the same way as capitalizing Copr even though it's not an abbreviation :D